resources/charts/stolon/templates/keeper-statefulset.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "stolon.fullname" . }}-keeper
+  labels:
+    app: {{ template "stolon.name" . }}
+    product: {{ template "stolon.name" . }}
+    chart: {{ template "stolon.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    stolon-keeper: "yes"
+    name: stolon-keeper
+spec:
+  serviceName: {{ template "stolon.fullname" . }}-keeper-headless
+  replicas: {{ .Values.keeper.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "stolon.name" . }}
+      release: {{ .Release.Name }}
+      component: stolon-keeper
+      name: stolon-keeper
+  template:
+    metadata:
+      labels:
+        app: {{ template "stolon.name" . }}
+        product: {{ template "stolon.name" . }}
+        release: {{ .Release.Name }}
+        component: stolon-keeper
+        stolon-cluster: {{ template "stolon.clusterName" . }}
+        stolon-keeper: "yes"
+        name: stolon-keeper
+    spec:
+      serviceAccountName: {{ template "stolon.serviceAccountName" . }}
+      {{- with .Values.keeper.nodeSelector }}
+      nodeSelector:
+        {{ toYaml . | nindent 8 }}
```

Comment:
Have you found out whether it's possible to use a fixed (i.e. relative) indent?

Reply:
There is no way to use relative indentations. :(